cmd/dagger/main.go

```diff
@@ -31,8 +32,8 @@ import (
 	"github.com/dagger/dagger/analytics"
 	"github.com/dagger/dagger/dagql/idtui"
 	"github.com/dagger/dagger/engine"
-	"github.com/dagger/dagger/telemetry"
-	"github.com/dagger/dagger/telemetry/sdklog"
+	enginetel "github.com/dagger/dagger/engine/telemetry"
```

Comment:
Could maybe rename this package, since it's used by both the CLI and the engine, but don't really know a better name. :thinking: This code is split out to minimize the `dagger.io/dagger/telemetry` dependency graph.